src/Creitive/Breadcrumbs/Breadcrumbs.php

```diff
@@ -350,6 +356,35 @@ public function setListElement($element)
 		$this->listElement = $element;
 	}
 
+/**
+     * Sets the icon breadcrumbs.
+     *
+     * @param string $classIcon
+     */
+    public function setIcon($classIcon)
+    {
+        if (!is_string($classIcon) && !is_null($classIcon))
+        {
+            throw new \InvalidArgumentException(
+                'Breadcrumbs::setIcon() only accepts strings or NULL, but '
+                . (is_object($classIcon) ? get_class($classIcon) : gettype($classIcon))
+                . ' given: ' . print_r($classIcon, true)
+            );
+        }
+
+        $this->icon = " <i class='{$classIcon}'></i> ";
```

Comment:
I know it's not the most readable, but the rest of the library uses string concatenation and double quoted attributes within the rendered output, so could you change this to use the same convention?

Also, I don't know about the space before the icon, maybe that kind of spacing should be handled by the end user via CSS? I don't have strong feelings about it either way.